liquibase-standard/src/test/java/liquibase/serializer/core/xml/XMLChangeLogSerializerTest.java

```diff
@@ -309,7 +309,7 @@ public void createNode_CreateIndexChange() throws Exception {
     public void createNode_CreateProcedureChange() throws Exception {
         CreateProcedureChange refactoring = new CreateProcedureChange();
         refactoring.setProcedureText("CREATE PROC PROCBODY HERE");
-        refactoring.setComments("Comments go here");
```

Comment:
Do you think we can add a new test for it?

Reply:
I'm not sure I understand what would be the new scenario we want to add.

Reply:
I mean that we are not really testing the comments here. We set them but we are not confirming that they are generated in `update-sql` command.

Reply:
Yeap, we removed that part because that comment is not being displayed anywhere. I am still not sure whether this comment should be in a procedure. If it's part of the body then it should be set as part of it, and then if we want to do some check against we should check whether a comment is contained or not as part of the body. If that's not the case, I don't know where it should be in an `update-sql` output.

Reply:
So we could change the description of the fields, as the help says:
```
  comment string 
    Description: Inline comments generated by update-sql. Not applied to the database
```
But checking comment [here](https://docs.liquibase.com/concepts/changelogs/changeset.html), it says: 
```
Specifies the description of the changeset. XML comments provide the same benefit.
```

I think we could use the same description and it would settle this issue for all.

cc @mariochampion